tests/unit/test_aws_bedrock_claude_chat_target.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import json
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from pyrit.models import (
+    ChatMessageListDictContent,
+    PromptRequestPiece,
+    PromptRequestResponse,
+)
+from pyrit.prompt_target.aws_bedrock_claude_chat_target import (
+    AWSBedrockClaudeChatTarget,
+)
+
+
+def is_boto3_installed():
+    try:
+        import boto3  # noqa: F401
+
+        return True
+    except ModuleNotFoundError:
+        return False
+
+
+@pytest.fixture
+def aws_target() -> AWSBedrockClaudeChatTarget:
+    return AWSBedrockClaudeChatTarget(
+        model_id="anthropic.claude-v2",
+        max_tokens=100,
+        temperature=0.7,
+        top_p=0.9,
+        top_k=50,
+        enable_ssl_verification=True,
+    )
+
+
+@pytest.fixture
+def mock_prompt_request():
+    request_piece = PromptRequestPiece(
+        role="user", original_value="Hello, Claude!", converted_value="Hello, how are you?"
+    )
+    return PromptRequestResponse(request_pieces=[request_piece])
+
+
+@pytest.mark.skipif(not is_boto3_installed(), reason="boto3 is not installed")
+@pytest.mark.asyncio
+async def test_send_prompt_async(aws_target, mock_prompt_request):
+    with patch("boto3.client", new_callable=MagicMock) as mock_boto:
+        mock_client = mock_boto.return_value
+        mock_client.invoke_model.return_value = {
+            "body": MagicMock(read=MagicMock(return_value=json.dumps({"content": [{"text": "I'm good, thanks!"}]})))
+        }
+
+        response = await aws_target.send_prompt_async(prompt_request=mock_prompt_request)
+
+        assert response.request_pieces[0].converted_value == "I'm good, thanks!"
+
+
+@pytest.mark.asyncio
+async def test_validate_request_valid(aws_target, mock_prompt_request):
+    aws_target._validate_request(prompt_request=mock_prompt_request)
+
+
+@pytest.mark.asyncio
+async def test_validate_request_invalid_data_type(aws_target):
```

Comment:
I think you need to "skipif" these two as well. 2 Tests are failing when we install just the core dependencies. I forget which ones but my money is on these two 🙂 
```suggestion
@pytest.mark.skipif(not is_boto3_installed(), reason="boto3 is not installed")
@pytest.mark.asyncio
async def test_validate_request_valid(aws_target, mock_prompt_request):
    aws_target._validate_request(prompt_request=mock_prompt_request)


@pytest.mark.skipif(not is_boto3_installed(), reason="boto3 is not installed")
@pytest.mark.asyncio
async def test_validate_request_invalid_data_type(aws_target):
```